frontend/src/locales/de.json

```diff
@@ -126,20 +114,41 @@
         "datetime": {
           "date": "dd. D.MM | D. MMM | DD. MMM YYYY"
         }
+      }
+    },
+    "print": {
+      "documents": {
+        "pdfDocument": {
+          "filename": {
+            "activitiesOnly": "Aktivitäten {camp}",
+            "picassoOnly": "Grobprogramm {camp}"
+          }
+        }
       },
-      "print": {
-        "documents": {
-          "pdfDocument": {
-            "filename": {
-              "activitiesOnly": "Aktivitäten {camp}",
-              "picassoOnly": "Grobprogramm {camp}"
-            }
+      "localPdfDownloadButton": {
+        "error": "Etwas hat nicht geklappt. Probiere es nochmals, oder lade die Seite neu."
+      },
+      "printConfigurator": {
+        "config": {
+          "Activity": "Einzelaktivität",
+          "Cover": "Titelseite",
+          "Picasso": "Grobprogramm",
+          "Story": "Roter Faden",
+          "Program": "Programm",
+          "Toc": "Inhaltsverzeichnis",
+          "ProgramConfig": {
+            "dayOverview": "Tagesübersicht drucken"
           }
-        },
-        "localPdfDownloadButton": {
-          "error": "Etwas hat nicht geklappt. Probiere es nochmals, oder lade die Seite neu."
         }
       },
+      "picasso": {
```

Comment:
Diese Ädnerung muss man bei react noch nachziehen.
![grafik](https://user-images.githubusercontent.com/1506818/162613166-44f04735-58c1-4a6c-b085-8a80f30048a2.png)

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/2562/commits/8a4c3c98252064d142a405b6932eb52f41411ff6